app-frontend/config/webpack/environments/production.js

```diff
@@ -6,6 +6,20 @@
 
 const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack_fail = function()

```

Comment:
Good catch. Ah that is probably useful then -- having `watch` fail hard on errors can be pretty annoying when developing. I don't feel super strongly, that makes it seem like using that library would be better then.